storage/src/vespa/storage/distributor/idealstatemetricsset.cpp

```diff
@@ -134,7 +134,7 @@ IdealStateMetricSet::IdealStateMetricSet()
 
 IdealStateMetricSet::~IdealStateMetricSet() = default;
 
-void IdealStateMetricSet::setPendingOperations(const std::vector<uint64_t>& newMetrics) {
+void IdealStateMetricSet::setPendingOperations(vespalib::ConstArrayRef<uint64_t> newMetrics) {
```

Comment:
For new code I think we should prefer `std::span<const T>` over `ConstArrayRef<T>`